nncf/tensor/functions/numeric.py

```diff
@@ -905,3 +905,42 @@ def ceil(a: Tensor) -> Tensor:
     :return: An array of the same type as a, containing the ceiling values.
     """
     return Tensor(ceil(a.data))
+
+
+@functools.singledispatch
+@tensor_guard
+def inverted_divide(a: Union[Tensor, float], b: Union[Tensor, float], invert: Optional[bool] = True) -> Tensor:
```

Comment:
Taking into account UX, I would suggest that NNCF should be aligned with numpy and torch. The `reciprocal` function is used to solve your task `a*(1.0 / b)` as follow: `a * reciprocal(b)`. I would suggest to introduce `reciprocal` function instead of `inverted_divide` and `inplace_inverted_divide`.

numpy: https://numpy.org/doc/2.1/reference/generated/numpy.reciprocal.html
torch: https://pytorch.org/docs/stable/generated/torch.reciprocal.html